src/main/webapp/stylesheets/repUserPassStyle.css

```diff
@@ -1,3 +1,3 @@
 h2{
     padding-top: 150px;
-}
\ No newline at end of file
+}
```

Comment:
Trailing line didn't save and now I can't commit the file again because no changes were detected lol